### DISCUSSION TITLE
Trucks back to the depot

### DISCUSSION THREAD

**Discussion by jdham89** [voxeltycoon/issues#487] [General] [answered]
is there a way to have the tucks that you send back to the depot drive there instead of stopping in the middle of the road and blocking the other traffic. It creates a backup of your other trucks when you select one to go back. Thanks!! the game is awesome!!

**Comment by victorleblais** [comment]
It could/should also apply to trains.

**Comment by victorleblais** [comment, marked as answer]
This feature is planned for Stage 1 of Steam Early Access: https://trello.com/c/91M7qVeH/19-moving-to-depot-w-o-teleportation
